tasks/main.yml

```
@@ -2,6 +2,8 @@
 
 - import_tasks: checkvars.yml
 
+- import_tasks: check-for-sessions.yml
```

Instead of checking on every task inside `check-for-sessions.yml` for `bbb_check_for_running_meetings` why not check for that here?
Like below for the `bbb_firewall_ufw['enabled']`.

```suggestion
- import_tasks: check-for-sessions.yml
  when: bbb_check_for_running_meetings
```